### PR TITLE
Feature/render votes from db

### PR DESCRIPTION
**Pull request by dghalbr**

### CODE REVIEW

File: src/component/VoteList.js [hunk 1]
```diff
@@ -1,9 +1,14 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import axios from 'axios';
 import { ListGroup, ListGroupItem, PageHeader } from 'react-bootstrap';
 
 class VoteList extends Component {
 
+  componentWillMount(){
+    this.props.getVotes();
```
Comment (neighbor983) [inline src/component/VoteList.js]: Move 'this.props.getVotes();' to 'componentDidMount()'. That is the preferred place to make API calls see [react docs](https://reactjs.org/docs/react-component.html#componentdidmount).

File: src/App.js [hunk 2]
```diff
@@ -18,18 +19,23 @@ class App extends Component {
...
-    this.getVoteList = this.getVoteList.bind(this);
-
   }
 
   changeMessage = (prevState) => {
     this.setState({ ...prevState, message: "blah, blah, blah" });
   };
 
+  getVotes = (prevState) => {
+    axios.post('/api/votes/votelist').then((data) => {
```
Comment (neighbor983) [inline src/App.js]: Just feels like this should be a 'get'

File: server/routes/vote.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+const router = express.Router();
+require('dotenv').config();
+
+const Vote = require('../models/vote');
+
+function createVote(requestBody) {
+    let newVote = new Vote({
+        id: requestBody.id,
```
Comment (neighbor983) [inline server/routes/vote.js]: There will no 'requestBody.id' at this point. An id will be created once the item is put into the db.

File: src/component/VoteList.js (hunk 1 above)
Comment (dghalbr) [reply src/component/VoteList.js]: Ok

File: src/App.js (hunk 2 above)
Comment (dghalbr) [reply src/App.js]: Yeah i didn't switch that out because it wasn't working, but now it is lol

File: server/routes/vote.js (hunk 3 above)
Comment (dghalbr) [reply server/routes/vote.js]: sure

**Pull request merged**